arch/x86/Makefile.in

```diff
@@ -103,25 +104,31 @@ adler32_avx2.o: $(SRCDIR)/adler32_avx2.c
 	$(CC) $(CFLAGS) $(AVX2FLAG) $(NOLTOFLAG) $(INCLUDES) -c -o $@ $(SRCDIR)/adler32_avx2.c
 
 adler32_avx2.lo: $(SRCDIR)/adler32_avx2.c
-	$(CC) $(SFLAGS) $(AVX2FLAG) $(NOLTOFLAG) $(INCLUDES) -c -o $@ $(SRCDIR)/adler32_avx2.c
+	$(CC) $(SFLAGS) $(AVX2FLAG) $(NOLTOFLAG) -DPIC $(INCLUDES) -c -o $@ $(SRCDIR)/adler32_avx2.c
```

Comment:
Why the `-DPIC` for this and the files below?

Reply:
It's actually important for the shared libraries, I think we were getting lucky with how those were being exported.  If I didn't put that define there (like they are in all the other SFLAG files above), the pkgcheck could fail comparing the cmake versus the configure.